stl/inc/new

```diff
@@ -94,7 +94,7 @@ _EXPORT_STD template <class _Ty>
 _NODISCARD_LAUNDER constexpr _Ty* launder(_Ty* _Ptr) noexcept {
     static_assert(!is_function_v<_Ty> && !is_void_v<_Ty>,
         "N4950 [ptr.launder]/1: Mandates: !is_function_v<T> && !is_void_v<T> is true.");
-    return __builtin_launder(_Ptr);
+    return ::__builtin_launder(_Ptr); // TRANSITION, DevCom-10456452, should not be qualified

```

Comment:
Since only MSVC is affected, we shouldn't apply the workaround to Clang and EDG.